dts/arm/nordic/nrf52832.dtsi

```diff
@@ -27,6 +27,7 @@
 				compatible = "soc-nv-flash";
 				label = "NRF_FLASH";
 				reg = <0x00000000 DT_FLASH_SIZE>;
+				erase-block-size = <1024>;
```

Comment:
Based on the discussion in #8705, should this be 4096?

Reply:
yeah, I guess so.